test/v1_integration.py

```diff
@@ -611,3 +611,10 @@ def test_sct_embedding():
         if abs(delta) > datetime.timedelta(hours=1):
             raise Exception("Delta between SCT timestamp and now was too great "
                 "%s vs %s (%s)" % (sct.timestamp, datetime.datetime.now(), delta))
+
+def test_auth_deactivation():
```

Comment:
I believe this only tests deactivating a pending authorization. Do you think there's value in also testing that a valid authorization can be deactivated?

Reply:
Done.

Reply:
Thanks!